catalog/openshift-cert-manager-operator/bundle.yaml

```diff
@@ -344,18 +344,18 @@ properties:
     provider:
       name: Red Hat
 relatedImages:
-- image: quay.io/redhat-user-workloads/cert-manager-oape-tenant/cert-manager-operator-1-15/cert-manager-operator-bundle-1-15@sha256:63bdca609bdc6fc41a4ed6deeb2317a702dd4200099026095c95ff1a77ac6e0e
-  name: ""
-- image: registry.redhat.io/cert-manager/cert-manager-operator-rhel9@sha256:bbf2fbcaa5751108a6babc0a74c509d35324994f7414f9293b9d8e6773404007
-  name: cert-manager-operator
-- image: registry.redhat.io/cert-manager/jetstack-cert-manager-acmesolver-rhel9@sha256:54a06ce6c31b6f3c1d7798e13e017e623ebfb5076d7810b7397be4ae59207427
-  name: cert_manager_acmesolver
-- image: registry.redhat.io/cert-manager/jetstack-cert-manager-rhel9@sha256:db40bfc8705ae82bbc38fbef95e0092891534dedd3ba903807399848decb94f5
-  name: cert_manager_webhook
-- image: registry.redhat.io/cert-manager/jetstack-cert-manager-rhel9@sha256:db40bfc8705ae82bbc38fbef95e0092891534dedd3ba903807399848decb94f5
-  name: cert_manager_ca_injector
-- image: registry.redhat.io/cert-manager/jetstack-cert-manager-rhel9@sha256:db40bfc8705ae82bbc38fbef95e0092891534dedd3ba903807399848decb94f5
-  name: cert_manager_controller
 - image: registry.redhat.io/openshift4/ose-kube-rbac-proxy-rhel9@sha256:6fbeaf03058bb5d6e9e3311a7afe67666e0770532c978b6b57e77ea2851cb085
   name: ""
+- image: registry.stage.redhat.io/cert-manager/cert-manager-operator-bundle@sha256:57f871eb24ac52cb091062b1bb30557d34f6d833d10809334e5c00b6ea9004ac
+  name: ""
+- image: registry.stage.redhat.io/cert-manager/cert-manager-operator-rhel9@sha256:83b98de77feed9df89e2dee8618e47e9801b383282636de785aa2746dfdd1ba2
```

Comment:
Bundle and operator image itself should not be part of relatedImages section.

Reply:
https://olm.operatorframework.io/docs/reference/file-based-catalogs/#olmbundle
^ referring to olm.bundle schema example, it seems bundle and operator images are part of the relatedImages stanza too, however please ensure that we use non-empty name(s) atleast. 

My bad, but to be clear we may not add these into ClusterServiceVersion's relatedImages.